core/jazz_create-serverless-service/index.js

```diff
@@ -92,7 +92,7 @@ var handler = (event, context, cb) => {
     // Validate and set deployment accounts
     let primaryAccountCount = 0;
 
-    if (Array.isArray(service_creation_data.deployment_accounts) && service_creation_data.deployment_accounts) {
+    if (service_creation_data.deployment_accounts && Array.isArray(service_creation_data.deployment_accounts) && service_creation_data.deployment_accounts) {
```

Comment:
Remove `service_creation_data.deployment_accounts` null check at the end.

Reply:
Array.isArray check is needed only if `service_creation_data.deployment_accounts` is not null - removing null check at the end.